openhrp3 couldnot compile if we remove build/CMakeCache.txt??

```
-- +++ processing catkin package: 'openhrp3'
-- ==> add_subdirectory(rtm-ros-robotics/openrtm_common/openhrp3)
CMake Error at rtm-ros-robotics/openrtm_common/openhrp3/catkin.cmake:96 (message):
  grep bin
  /home/k-okada/catkin_ws/ws_rtmros_gazebo/build/rtm-ros-robotics/openrtm_common/openhrp3/build/OpenHRP-3.1/install_manifest.txt
Call Stack (most recent call first):
  rtm-ros-robotics/openrtm_common/openhrp3/CMakeLists.txt:2 (include)
```

--- a/catkin.cmake
+++ b/catkin.cmake
@@ -9,6 +9,9 @@ find_package(catkin REQUIRED COMPONENTS mk rostest openrtm_aist)
 # <devel>/lib/<package>/bin/openrhrp-*
 # <devel>/lib/libhrp...
 # <src>/<package>/share
+if(NOT openhrp3_FOUND)
+  file(REMOVE ${CMAKE_CURRENT_BINARY_DIR}/installed)
+endif()
 if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/installed)
 
   set(ENV{PATH} ${openrtm_aist_PREFIX}/lib/openrtm_aist/bin/:$ENV{PATH}) #update PATH for rtm-config

